pkgs/by-name/ch/chroot-realpath/package.nix

```diff
@@ -0,0 +1,21 @@
+{
+  lib,
+  rustPlatform,
+}:
+
+let
+  cargo = builtins.fromTOML (builtins.readFile ./Cargo.toml);
+in
+rustPlatform.buildRustPackage {
+  pname = cargo.package.name;
+  version = cargo.package.version;
+
+  src = ./.;
```

Comment:
I think we prefer putting sources in a source dir so that for instance the .nix file doesn't cause pointless rebuilds.

Reply:
This is why I wanted to use filesets but we're not allowed :( I'll move things to `src`